### PR TITLE
Add binding to get DOT string of control flow graph 

### PR DESCRIPTION
**Pull request by sklam**
Alternative for https://github.com/numba/llvmlite/pull/64 because the `viewCFG` api from llvm dumps the DOT file to random temp file and tries to open it with `gv`.  Instead, this PR gets the DOT string of the CFG and returns it.

### CODE REVIEW

**Comment by sklam** [comment]
This addition will be useful for debugging in numba.

File: llvmlite/tests/test_binding.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from llvmlite import six, ir
 from llvmlite import binding as llvm
 from llvmlite.binding import ffi
+from llvmlite import ir
```
Comment (pitrou) [inline llvmlite/tests/test_binding.py]: The import is already done three lines above.

**Comment by pitrou** [comment]
Ok, I merged despite the tiny nit mentioned above. This can be fixed in a later changeset if we care enough.

**Comment by sklam** [comment]
Oops, thanks

**Comment by sklam** [comment]
@pitrou, I cleaned up some dead import at https://github.com/numba/llvmlite/pull/204/files

**Pull request merged**